.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,76 @@
+name: Bug Report
+description: Report an Insomnia bug
+labels: [B-bug, S-unverified]
+body:
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+      - label: I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for a bug that matches the one I want to file, without success.
+        required: true
+- type: input
+  attributes:
+    label: Insomnia Version
+    description: What version of Insmonia are you using?
+    placeholder: "2021.5"
```

```suggestion
    placeholder: "2021.5.0"
```

There is no version of insomnia that matches this - it seems best to not avoid confusion and imply that people use actual version numbers.